package.json

```diff
@@ -33,18 +33,20 @@
     }
   ],
   "bin": "cli.js",
+  "typings": "types/generated.d.ts",
   "engines": {
     "node": ">=0.10.0"
   },
   "scripts": {
     "test": "xo && nyc --cache --reporter=lcov --reporter=text tap --no-cov --timeout=150 test/*.js test/reporters/*.js",
     "test-win": "tap --no-cov --reporter=classic --timeout=150 test/*.js test/reporters/*.js",
-    "visual": "node test/visual/run-visual-tests.js"
+    "visual": "node test/visual/run-visual-tests.js",
+    "make-ts": "node types/make"
```

Comment:
This should probably be a `prepublish` script. Otherwise I am going to forget this step when publishing.